### DISCUSSION TITLE
Multiplayer

### DISCUSSION THREAD

**Discussion by corpsequest** [directvt/vtm#480] [Q&A]
i cannot for the life of me figure out how one is meant to share a session with other users.
this is the most progress i've made: `os: sock: Foreign users are not allowed to the session (22)`

**Comment by o-sdn-o** [comment]
Until recently, I used the wrong wording - Multiplayer. In one of the latest updates, I tried to fix this in the [Process Model](https://github.com/directvt/vtm/blob/master/doc/architecture.md) section in `/vtm/doc/architecture.md`.

The idea is that a session created/launched by a user allows multiple connections only for creator or users with elevated privileges.

Unfortunately, I have not corrected the wording in all places where this is mentioned.

This comes from the days when a demo server was publicly available, to which anyone could connect simultaneously in real time. In fact, everyone accessed this server via SSH under one fixed local account.

For the test you can run several vtm in different terminals on your system. Connection via SSH to Windows is waiting for a fix #464.

**Reply by corpsequest** [reply]
so if i make a user and simply share it between myself and someone else, this achieves the same effect? that sounds right but i feel i should ask for the sake of not letting assumption kill me

**Reply by o-sdn-o** [reply]
Users have to use
```
runuser -u user -- /path/to/vtm
```
or
```
su - user -c /path/to/vtm
```
or remotely
```
ssh user@vtm_running_host
```

to connect to the session created for the user you shared (on unix).

**Reply by o-sdn-o** [reply]
It works well on Linux. What operating system are you using?

**Reply by o-sdn-o** [reply]
I suspect that vtm may have some bugs related to platform-specific sandboxing on other platforms.
